staging/script.xsqueeze/resources/settings.xml

```diff
@@ -45,4 +45,19 @@
        <setting label="(but if it is too low you may experience crashes crashes)" type="lsep"/>
        <setting id="waitOnAS" type="number" label="Max wait seconds for artist slideshow to exit cleanly... (default 10)" default="10"/>
     </category>
+    <category label="HDMI CEC">
+       <setting label="Please go to http://kodi.wiki/view/Add-on:XSqueeze and read about how to" type="lsep"/>
+       <setting label="install the required dependencies." type="lsep"/>
```

Comment:
If the dependency is simple can you inline the advice here?  Also, is the cec thing you use standard with, say openelec?